README.md

```diff
@@ -1,7 +1,24 @@
 # CloudAI Benchmark Suite
 
 ## Project Description
-CloudAI benchmark suite aims to develop an industry standard benchmark focused on grading Data Center (DC) scale AI systems in the Cloud.
+CloudAI benchmark suite aims to develop an industry standard benchmark focused on grading Data Center (DC) scale AI systems in the Cloud. The primary motivation is to provide automated benchmarking on various systems, including Slurm, Kubernetes (K8S), and proprietary systems. Currently, CloudAI supports Slurm with enroot.
```

Comment:
Let's remove this - ", including Slurm, Kubernetes (K8S), and proprietary systems." Also in supported systems below.

Reply:
Updated.